imageproxy.go

```diff
@@ -203,6 +211,20 @@ func validHost(hosts []string, u *url.URL) bool {
 	return false
 }
 
+// blacklistedHost returns true if the host is in our blacklist
+func blacklistedHost(hosts []string, u *url.URL) bool {
```

Comment:
is this an exact copy of `validHost` or am I overlooking a subtle difference.  If it is exact, then why duplicate it?  since it is now being used for both whitelist and blacklist, we could rename it to something more neutral like `hostMatches` or something if the naming was the concern.